Makefile

```diff
@@ -72,30 +80,27 @@ test: venv-tools
 		then venv-tools/bin/pytest ./**/*.py -vv ; \
 	fi
 
-.PHONY: format
+.PHONY: format setup-web
 format: venv-tools
 	venv-tools/bin/ruff check --fix --unsafe-fixes
 	venv-tools/bin/ruff format
+	yarn run format
 
 .PHONY: format-check
 format-check: venv-tools
 	venv-tools/bin/ruff check
 	venv-tools/bin/ruff format --check
 
```

Comment:
Does `yarn run format` have a flag for `check` that we can use in here so that the pipeline can check frontend formatting?

Reply:
For CI,  the task was format:verify